docs/usage.rst

```diff
@@ -5,20 +5,73 @@ Usage
 Authentication
 --------------
 
-To access an Artifactory server,
-store your username and `API key`_ in :file:`~/.artifactory_python.cfg`
+Artifactory servers can allow anonymous access
+by logging in with a fixed pair
+of ``'anonymous'`` as username
+and ``''`` as password.
+This is the default behavior of :mod:`audfactory`.
+
+.. jupyter-execute::
+
+    import audfactory
+
+    audfactory.authentification('https://artifactory.domain.com/artifactory')
+
+To access an Artifactory server
+that requires logging in with a username and password,
+store your username and `API key`_
+in :file:`~/.artifactory_python.cfg`
+using separate sections for every server.
+Every section is marked
+by the server URL in square brackets
+without the ``https://`` or ``http://``
+at the beginning:
 
 .. code-block:: cfg
 
-    [artifactory.audeering.com/artifactory]
-    username = MY_USERNAME
-    password = MY_API_KEY
+    [artifactory1.domain.com/artifactory]
+    username = MY_USERNAME1
+    password = MY_API_KEY1
+
+    [artifactory2.domain.com/artifactory]
+    username = MY_USERNAME2
+    password = MY_API_KEY2
+
+.. jupyter-execute::
+    :hide-code:
+    :hide-output:
+
+    import os
+
+    os.environ['ARTIFACTORY_USERNAME'] = 'MY_USERNAME2'
+    os.environ['ARTIFACTORY_API_KEY'] = 'MY_API_KEY2'
+
+When request to authenticate for a configured server URL,
```

Comment:
Probably you want 
"When authentication is requested for a configured server URL, ..." or simular

Reply:
Changed the text accordingly.